apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -633,9 +716,26 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): J
                             || schema.name === ProfileConstants?.SCIM2_SCHEMA_DICTIONARY.get("ACCOUNT_DISABLED")
                             || schema.name === ProfileConstants?.SCIM2_SCHEMA_DICTIONARY.get("ONETIME_PASSWORD"))) {
                             return (
-                                <List.Item key={ index } className="inner-list-item">
-                                    { generateSchemaForm(schema) }
-                                </List.Item>
+                                <>
+                                    {
+                                        showMobileUpdateWizard && checkSchemaType(schema.name, "mobile")
+                                            ? (
+                                                < MobileUpdateWizard
+                                                    onAlertFired={ onAlertFired }
+                                                    closeWizard={ () =>
+                                                        handleCloseMobileUpdateWizard()
+                                                    }
+                                                    wizardOpen={ true }
+                                                    currentMobileNumber={ profileInfo.get(schema.name) }
+                                                    isMobileRequired={ schema.required }
+                                                />
+                                            )
+                                            : null
+                                    }
+                                    <List.Item key={ index } className="inner-list-item">
+                                        {generateSchemaForm(schema)}
```

Comment:
```suggestion
                                        { generateSchemaForm(schema) }
```

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1275/commits/375a1a7a8b040739aaae46cb9b9657e461a0bf70